docs/source/en/model_doc/rwkv.mdx

```diff
@@ -64,43 +64,43 @@ torch.allclose(torch.cat([output_one, output_two], dim=1), output_whole, atol=1e
 
 ## Rwkv attention and the recurrent formulas
 
-In a traditional auto-regressive Transformer, attention is written as
+In a traditional autoregressive Transformer, attention is written as
 
 $$O = \hbox{softmax}(QK^{T} / \sqrt{d}) V$$
 
-with \\(Q\\), \\(K\\) and \\(V\\) are matrices of shape `seq_len x hidden_size` named query, key and value (they are actually bigger matrices with a batch dimension and an attention head dimension but we're only interested in the last two, which is where the matrix product is taken, so for the sake of simplicity we only consider those two). The product \\(QK^{T}\\) then has shape `seq_len x seq_len` and we can take the maxtrix product with \\(V\\) to get the output \\(O\\) of the same shape as the others.  
+where \\(Q\\), \\(K\\) and \\(V\\) are matrices of shape `seq_len x hidden_size` named the Query, Key, and Value (in reality, these are matrices with higher dimensionality to account for a batch dimension and an attention head dimension. In this instance, we are only interested in the last two dimensions, which is where the matrix product is taken, so for the sake of simplicity we only consider those two). The product \\(QK^{T}\\) then has shape `seq_len x seq_len` and we can take the maxtrix product with \\(V\\) to get the output \\(O\\) of the same shape as the others.  
 
 Replacing the softmax by its value gives:
 
 $$O_{i} = \frac{\sum_{j=1}^{i} e^{Q_{i} K_{j}^{T} / \sqrt{d}} V_{j}}{\sum_{j=1}^{i} e^{Q_{i} K_{j}^{T} / \sqrt{d}}}$$
 
-Note that the entries in \\(QK^{T}\\) corresponding to \\(j > i\\) are masked (the sum stops at j) because the attention is not allowed to look at future tokens (only past ones).
+Note that the entries in \\(QK^{T}\\) corresponding to \\(j > i\\) are masked (the sum stops at \\(j)\\) because the attention is not allowed to look at future tokens (only past ones).
```

Comment:
I think we need to end the latex formatting on the inner bracket here

```suggestion
Note that the entries in \\(QK^{T}\\) corresponding to \\(j > i\\) are masked (the sum stops at \\(j\\)) because the attention is not allowed to look at future tokens (only past ones).
```